api/draft_registrations/serializers.py

```diff
@@ -142,6 +151,7 @@ class DraftRegistrationDetailSerializer(DraftRegistrationSerializer, DraftRegist
     Overrides DraftRegistrationLegacySerializer to make id required.
     registration_supplement, node, cannot be changed after draft has been created.
     """
+    id = IDField(source='_id', required=True)
```

Comment:
Not sure why we need this one since both `DraftRegistrationSerializer` and `DraftRegistrationDetailLegacySerializer` defines the `id` the same way with `id = IDField(source='_id', required=True)`. But it doesn't hurt.

Reply:
Update: I was wrong. One parent/grandparent has id = IDField(source='_id', required=True) while another has `id = IDField(source='_id', read_only=True)`. We have to explicitly define the `id` here.